pkg/client/client.go

```diff
@@ -1066,18 +1066,18 @@ func (c *immuClient) Dump(ctx context.Context, writer io.WriteSeeker) (int64, er
 		}
 		if err != nil {
 			errs = append(errs, fmt.Sprintf("error receiving chunk: %v", err))
-			continue
+			break
```

Comment:
If we break the loop everywhere, instead of continuing it, then gathering errors does not make sense anymore.
In fact, if we break the loop on the first error, then it makes more sense to just return the error immediately - i.e. return, instead of break.
What do you think, @Gjergj ?

Reply:
I agree. Still I'd like to keep the same flow to have the same structure and messages appear to the user.

Reply:
Why?

Reply:
As in the following commit

Reply:
If the error happens in the inner `for`, the outer `for` will still run (and also any subsequent inner `for`s) => the error will be overwritten. Is this intended behaviour?

Reply:
From the original code I assumed that the reason for the continue usage was that no matter what error still continue dumping data.
How should dump behave on error abort immediately or continue no matter what?